src/services/AuthService.ts

```diff
@@ -31,4 +78,25 @@ export class AuthService {
       return userRepository.deleteUser(user);
     });
   }
+
+  public async getSessionsByUserId(id: Uuid): Promise<APIUserSession[]> {
+    return this.transactions.readOnly(async (transactionalEntityManager) => {
+      const sessionRepository = Repositories.session(transactionalEntityManager);
+      const sessions = await sessionRepository.getSessionsByUserId(id);
+      if (!sessions) throw new NotFoundError('User not found!');
+      const apiSessions: APIUserSession[] = [];
+      sessions.forEach(function (session) {
+        apiSessions.push(session.serializeToken());
+      });
+      return apiSessions;
+    });
+  }
+
+  public async deleteSessionByAccessToken(accessToken: string): Promise<boolean> {
+    return this.transactions.readWrite(async (transactionalEntityManager) => {
+      const sessionRepository = Repositories.session(transactionalEntityManager);
+      const success = await sessionRepository.deleteSessionByAccessToken(accessToken);
+      return success;
```

Comment:
can't i just return the line above?

Reply:
ya i did this just for readability - that we're returning if it's success or not, and so people who work on this in the future know instantly what the output of `deleteSessionByAccessToken` is